circle.yml

```diff
@@ -15,8 +15,8 @@ dependencies:
 
 test:
   override:
-    - tox -e flake8
```

Comment:
No point splitting up the `tox` call if you're not going to pass any args in or anything